### ISSUE TITLE
Mysterious iframe (?) error

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#658]
AMP is reporting a number of violations around this HTML that it found on 118 pages:

`<html><head><link rel="alternate stylesheet" type="text/css" href="resource://content-accessible/plaintext.css" title="Wrap Long Lines"></head><body><pre>Retry later</pre><script src="moz-extension://88ca6e4b-1c00-46cb-a9f9-56f6c66a5aae/AccessEngine.professional.js"></script></body></html>`

Errors:
- This DOM does not contain a document title element
- This HTML does not have a lang attribute
- The body element does not contain elements that form a heading structure (i.e. h1-h6 elements; or elements given a role=heading)

Example pages:
https://earthworks.stanford.edu/catalog/stanford-zb910wv5967/web_services
https://earthworks.stanford.edu/catalog/harvard-us-cd102/email
https://earthworks.stanford.edu/catalog/ark28722-s7059k
https://earthworks.stanford.edu/catalog/facet/dc_format_s?facet.sort=index&featured=california_data&q=

The only thing the pages have in common is that they're not search results. I can't find the reported HTML in the pages, but I'm wondering if it has anything to do with reCaptcha in the Feedback form. Same error does not occur in SearchWorks.

???

**Comment by mejackreed** [comment]
@jvine is it possible that this is coming from a Firefox extension that is installed? I'm not able to find this markup in the HTML?

**Comment by jvine** [comment]
I'm not able to find it either. I don't think it's a plugin issue, because it's found in the audit report (scripted process that doesn't use my browser) but not found in the in-browser plugin report on the same page. And the error has not appeared in any other asset's audit. 

Really weird. But I'm going to close the ticket.
